po/da.po

```diff
@@ -14899,14 +14899,17 @@ msgstr "Officiel dokumentation"
 
 #: src/other-resources.md:8
 msgid "The Rust project hosts many resources. These cover Rust in general:"
-msgstr ""
+msgstr "Rustprojektet hoster mange resourcer. Disse dækker Rust generelt:"
 
 #: src/other-resources.md:10
 msgid ""
 "[The Rust Programming Language](https://doc.rust-lang.org/book/): the "
 "canonical free book about Rust. Covers the language in detail and includes a "
 "few projects for people to build."
 msgstr ""
+"[The Rust Programming Language](https://doc.rust-lang.org/book/): en gratis "
+"bog om Rust. Dækker sproget i detaljer og indeholder et par projekter du kan "
```

Comment:
"Dækker sproget grundigt" or "Dækker sproget detaljeret"?

Reply:
Sure, let's try the second suggestion:

```suggestion
"bog om Rust. Dækker sproget detaljeret og indeholder et par projekter du kan "
```